src/data-publication/reports/tables/Aggregate1.jsx

```diff
@@ -59,99 +105,172 @@ const Aggregate1 = React.forwardRef((props, ref) => {
     return 0
   })
 
-  let colWidth = '5.7%'
+  return fixUnaggregatedDispositions(sorted)
+}
+
+export const buildCSVRowsAggregate1 = (report) => {
+  let theCSVRows = ''
+
+  const _data = sortAndFix(report)
+  
+  _data.forEach(tract => {
+    // Tract header
+    theCSVRows += tract.tract + '\n'
+
+    // Disposition rows
+    tract.dispositions.sort(function(x,y){
+      var xp = x.titleForSorting.substring(x.titleForSorting.length-3);
+      var yp = y.titleForSorting.substring(y.titleForSorting.length-3);
+      return xp == yp ? 0 : xp < yp ? -1 : 1;
+    }).forEach(dispo => {
+      const _rowLabel = dispo.title
+      const _cols = dispo.values.sort(function(x,y){
+        var xp = x.dispositionName.substring(x.dispositionName.length-3);
+        var yp = y.dispositionName.substring(y.dispositionName.length-3);
+        return xp == yp ? 0 : xp < yp ? -1 : 1;
+      }).map(dispo => [dispo.count, dispo.value]).flat().join(',')
+
+      theCSVRows += _rowLabel + ',' + _cols + '\n'
+    })
+  })
+
+  return theCSVRows
+}
+
+const formatDisplayLabel = (items) => {
```

Comment:
Could we move the `formatDisplayLabel` function to a utility file? We are using it in both `Aggregate` table files.

Reply:
Done.